server/src/test/java/io/spine/server/bc/BoundedContextShould.java

```diff
@@ -337,4 +350,61 @@ public void do_not_expose_invisible_aggregate() {
         assertFalse(boundedContext.findRepository(SecretProject.class)
                                   .isPresent());
     }
+
+    /**
+     * In the process of registering {@link Repository} in the {@link BoundedContext},
```

Comment:
```
This test checks, whether {@code BoundedContext} properly handles the issues upon repository registration.

<p>In particular, we intentionally break  an interaction between {@code Model} and a {@code BoundedContext} on attempt to ensure there is an entity default state present for the given repository instance. 

<p>The expected behavior of {@code BoundedContext} instance is to fail fast in case such a default state is absent. 

<p>In real-life this use case can never happen given the current implementation of {@code Model} and {@code BoundedContext}. However, previously such an issue was caught. Therefore this test case ensures it's never happening again.
```